books/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# -- Build stage --
+
+FROM registry.redhat.io/ubi8/go-toolset:1.12.8 AS builder
+
+WORKDIR $HOME/go/src/books
+COPY *.go .
+
+RUN go get -u \
+      github.com/gorilla/mux \
+      github.com/lib/pq && \
+    go build
+
+
+# -- App stage --
```

Comment:
A multi stage Dockerfile seems a little much for this application. 
I would suggest to add OpenShift labels, with them OpenShift will create the service for the application for free, check this Dockerfile for reference: https://github.com/RedHatTraining/DO280-apps/blob/master/famous-quotes/Dockerfile#L8